src/tennisSimulator/Serve.java

```diff
@@ -0,0 +1,46 @@
+package tennisSimulator;
+
+public class Serve {
+
+    private final Player firstPlayer;
+    private final Player secondPlayer;
+    private final Double SERVING_FACTOR = 0.2;
```

Comment:
If there is no need to use Double class functionality, then use primitive type double